context/context.go

```diff
@@ -36,11 +36,11 @@ func ResolveRemotesToRepos(remotes Remotes, client *api.Client, base string) (*R
 	return result, nil
 }
 
-func resolveNetwork(result *ResolvedRemotes) error {
+func resolveNetwork(result *ResolvedRemotes, maxRemotesForLookup int) error {
 	var repos []ghrepo.Interface
 	for _, r := range result.remotes {
 		repos = append(repos, r)
-		if len(repos) == maxRemotesForLookup {
+		if len(repos) == maxRemotesForLookup || maxRemotesForLookup == 0 {
```

Comment:
This line can remain unchanged since we append to `repos` before this check. If `remotesForLookup` is 0 then it will never match the break condition.